src/pages/about.js

```diff
@@ -1,10 +1,99 @@
-import * as React from 'react';
-import Layout from '../components/Layout';
-
-const About = () => (
-  <Layout>
-    <h1>This is the about page!</h1>
-  </Layout>
-);
+import { graphql, useStaticQuery } from "gatsby";
+import * as React from "react";
+import Layout from "../components/Layout";
+import CallOut from "../components/Callout";
+import {
+  AboutDescription,
+  AboutTitle,
+  Section,
+  ReverseSection,
+  AboutImages,
+} from "../styles/about.styles";
+import { RichText } from "prismic-reactjs";
+
+import { H1, H2, P, MarginContainer } from "../styles/styles";
+
+const About = () => {
+  const aboutQuery = useStaticQuery(graphql`
+    query AboutQuery {
+      prismicAboutPage {
+        data {
+          callout_cta_label {
+            text
+          }
+          callout_heading {
+            text
+          }
+          main_heading {
+            text
+          }
+          sections {
+            image {
+              url
+            }
+            section_header {
+              text
+            }
+            section_text {
+              richText
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  const {
+    prismicAboutPage: {
+      data: {
+        main_heading: mainHeading,
+        sections,
+        callout_heading: calloutHeading,
+        callout_cta_label: calloutLabel,
+      },
+    },
+  } = aboutQuery;
+
+  return (
+    <Layout>
+      <MarginContainer>
+        <AboutTitle>
+          <H1>{mainHeading.text}</H1>
+        </AboutTitle>
+        <Section>
+          <AboutDescription>
+            <H2 bold>{sections[0].section_header.text}</H2>
+            <P>
+              <RichText render={sections[0].section_text.richText} />
+            </P>
```

Comment:
`RichText` will render a `p` tag and nested `a` tags so you don't need to wrap it in a `P` tag! you can use a `div` if you need to style